README.md

```diff
@@ -1,106 +1,135 @@
+     ```bash
+     sudo docker compose up
+     ```
+
+   Use the `-d` flag to run the application in daemon mode:
+   ```bash
+   sudo docker compose up -d
+   ```
+
+   **Note:** When starting the application for the first time, it takes some time to initialize the ArduPilot Git repositories at the backend. This process also involves populating the list of available versions and releases using the GitHub API, so please be patient.
+
+5. **Access the Web Interface:** 
+   The application binds to port 11080 on your host machine by default. Open your web browser and go to `http://localhost:11080` to interact with the web interface.
+
+6. **Stopping the Application:**
+   To stop the application, you can use the following command:
+   ```bash
+   sudo docker compose down
+   ```
+   This will stop and remove the containers, but it will not delete any built images or volumes, preserving your data for future use.
+
+## Running Locally Without Docker on Ubuntu
+To run the ArduPilot Custom Firmware Builder locally without Docker, ensure you have an environment capable of building ArduPilot. Refer to the [ArduPilot Environment Setup Guide](https://ardupilot.org/dev/docs/building-setup-linux.html) if necessary.
+
+1. **Clone the Custom-Build Repository:**
+   ```bash
+   git clone https://github.com/ardupilot/CustomBuild.git
+   cd CustomBuild
+   ```
+2. **Create and use a virtual environment:**
+   ```bash
+   python3 -m venv path/to/virtual/env
+   source path/to/virtual/env/bin/activate
+   ```
+
+   If the python venv module is not installed, run:
+   ```bash
+   sudo apt install python3-venv
+   ```
+
+3. **Install Dependencies:**
+   ```bash
+   pip install -r web/requirements.txt -r builder/requirements.txt
+   ```
+
+   If pip is not installed, run:
+   ```bash
+   sudo apt install python3-pip
+   ```
+
+4. **Install and Run Redis:**
+   Use your package manager to install Redis:
+   ```bash
+   sudo apt install redis-server
+   ```
+   Ensure the Redis server is running:
+   ```bash
+   sudo systemctl status redis-server
+   ```
+
+5. **Execute the Application:**
+   - For a development environment, run:
+     ```bash
+     ./web/app.py
+     ```
+   - For a production environment, use:
+     ```bash
+     gunicorn web.wsgi:application
+     ```
+
+6. **Access the Web Interface:**
+   Once the application is running, it will display a local host port (default is 5000) in the terminal where the interface can be accessed. Open your web browser and go to `http://localhost:<displayed-port>` to interact with the application.
+
+The application will automatically set up the required base directory at `./base` upon first execution. You may customize this path using the `--basedir` option or by setting the `CBS_BASEDIR` environment variable. You can also pre-provision the contents of the base directory using the directory structure provided below to bootstrap the startup process.
```

Comment:
explain bootstrap the startup process

Reply:
I have removed this line completely. Let us not confuse users. Even if they setup the base directory on their own, it doesn't help in any way. It would take similar time if not more.